node-tests/acceptance/exam-test.js

```diff
@@ -293,6 +294,18 @@ describe('Acceptance | Exam Command', function() {
       unlinkFiles.push(testExecutionJsonPath);
     }
 
+    function assertModuleDetailJson(output) {
+      let moduleRunDetailJsonPath = path.join(
+        process.cwd(),
+        output.match(/module-metadata-([0-9]*).json/g)[0]
+      );
+      assert.ok(
+        fs.existsSync(moduleRunDetailJsonPath),
+        'module run detail json written to root'
+      );
+      unlinkFiles.push(moduleRunDetailJsonPath);
```

Comment:
if the assert fails.. this line won't get executed? so it may be better to move this above the assert.